src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -154,18 +155,33 @@ private string GetNextDataFolderName(string folderName = null)
             return dataFolderName;
         }
 
-        public void StartAll()
+        public void StartAll(bool enableLogs = false)
         {
             foreach (CoreNode node in this.Nodes.Where(n => n.State == CoreNodeState.Stopped))
             {
                 node.Start();
             }
+
+            if (enableLogs)
+            {
+                while (!LogManager.IsLoggingEnabled())
```

Comment:
why do we need while loop? 
If you call it once it doesn't work?

Reply:
@noescape00 internally it uses counter, look at Slack conversation

Reply:
Its just in case, if you look in the code there is a logs counter and the method `DisableLogging()` decrease the counter by 1

Reply:
Ah, ok. Makes perfect sense